packages/wdio-crossbrowsertesting-service/src/launcher.ts

```diff
@@ -7,7 +7,12 @@ import logger from '@wdio/logger'
 const log = logger('@wdio/crossbrowsertesting-service')
 
 export default class CrossBrowserTestingLauncher {
-    constructor (options, caps, config) {
+    options: WebdriverIO.ServiceOption;
+    config: WebdriverIO.Config;
+    tunnel: boolean = false;
+    cbtTunnelOpts: CBTConfigInterface = undefined as any;
+
+    constructor (options: WebdriverIO.ServiceOption, caps: WebDriver.DesiredCapabilities, config: WebdriverIO.Config) {
         this.options = options
         this.config = config
```

Comment:
Defining the constructor variables like this allows us to remove the definition from the top class header

```suggestion
    constructor (
        private _options: WebdriverIO.ServiceOption,
        private _caps: WebDriver.DesiredCapabilities,
        private _config: WebdriverIO.Config
    ) {}
```

Reply:
Done